_includes/sidebar.html

```diff
@@ -32,7 +32,7 @@ <h3>Contact</h3>
                     </a></li>
                 {% endif %}
                 {% if site.email %}
-                    <li><a aria-label="My Email" href="{{ site.email }}">
+                    <li><a aria-label="My Email" href="mailto:{{ site.email }}">
```

Comment:
Same could happen in your [another theme template](https://github.com/byanko55/jekyll-professional-resume).

Reply:
It's like you said.